### PR TITLE
Improve style of quote blocks

### PR DESCRIPTION
**Pull request by eternoendless**
| Questions     | Answers
| ------------- | -------------------------------------------------------
| Description?  | Improved style for quote blocks + fixed all wrong use of quote blocks where it was actually a notice.
| Fixed ticket? | ~

Before:
Screenshot 2020-07-10 at 17 57 47

After:
Screenshot 2020-07-10 at 17 58 04

### CODE REVIEW

File: src/themes/hugo-theme-learn/_src/sass/components/_blockquote.scss [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+  }
+
+  ul, ol {
+    margin: {
+      top: .5rem;
+      bottom: .5rem;
+    }
+
+    li {
+      margin-bottom: 0 !important;
```
Comment (matks) [inline src/themes/hugo-theme-learn/_src/sass/components/_blockquote.scss:27]: I'm very bad at html/css but I was always told never to use `!important` 😬 😄

**Review by matks — APPROVED** [review]
Code seems good but needs to be tested. We recently had issues with @NeOMakinG about pushing assets to be acknowledged by GitHub Pages.

File: src/themes/hugo-theme-learn/_src/sass/components/_blockquote.scss (hunk 1 above)
Comment (NeOMakinG) [reply src/themes/hugo-theme-learn/_src/sass/components/_blockquote.scss:27]: Well, did you try to select it a bit more precisely or using the selector causing you to be forced to use an important ?

**Comment by NeOMakinG** [comment]
> Code seems good but needs to be tested. We recently had issues with @NeOMakinG about pushing assets to be acknowledged by GitHub Pages.

Pushing CSS is working fine, JS was not, but JS is now builded inside travis before serving ! No more problems about this

Also, we could kick assets from this repository as I think CSS assets are also builded by travis

**Review by NeOMakinG — APPROVED** [review]

File: src/themes/hugo-theme-learn/_src/sass/components/_blockquote.scss (hunk 1 above)
Comment (eternoendless) [reply src/themes/hugo-theme-learn/_src/sass/components/_blockquote.scss:27]: I just copied and pasted the original css from "notice". That said, the `important` is probably justified, as there was probably another rule messing with `li` attached to a classnamed parent element, which takes precedence.

The whole theme would clearly benefit form a revamp/

**Review by PierreRambaud — APPROVED** [review]

**Comment by PierreRambaud** [comment]
Thanks @eternoendless

**Pull request merged**